src/installer.js

```diff
@@ -127,7 +127,7 @@ var getDefaults = function (data, callback) {
         // Read/write home directory access
         '--filesystem=home',
         // Chromium uses a socket in tmp for its singleton check
-        '--filesystem=/tmp',
+        '--env=TMPDIR=/var/tmp',
```

Comment:
This seems unrelated, but it does seem to make sense, better than providing full access to `/tmp`.